regulations/templates/regulations/comment.md

```diff
@@ -0,0 +1,12 @@
+{% spaceless %}
```

Comment:
Also asking for opinions about this in #django, but django templates might not be the best choice for building markdown. This produces a bit of extra whitespace, which can be cleaned up using some inelegant hacks like deleting newlines or writing something like

```
{% endfor %}{% comment %}
{% endcomment %}{% endfor %}
```

@vrajmohan @cmc333333